src/main/java/HomeWorks/HW4/example2/SpringApplicationAnimal.java

```diff
@@ -0,0 +1,11 @@
+package HomeWorks.HW4.example2;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringApplicationAnimal {
+    public static void main(String[] args) {
+        SpringApplication.run(SpringApplicationAnimal.class, args);
```

Comment:
why do you use to SpringApplicationMain classes? One is enough for us in the spring application

Reply:
fixed